Updated Scurrius

changednamedes opened the pull request:
Added more config options for food and prayer

Options to select which food, with quantity to withdraw - same with prayer

Fixed the prayers to be used correctly and disable/enable correctly to preserve prayer

fixed dodging of the dangerous tiles as this wasn't working

Added better checking for states, as well as checks for being in the room etc

changednamedes committed:
Updated Scurrius

Added more config options for food and prayer, with min and max to use

Fixed the prayers to be used correctly and disable/enable correctly

fixed dodging of the dangerous tiles

gmason0 commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/scurrius/ScurriusConfig.java:
```diff
@@ -1,18 +1,161 @@
+            position = 0
+    )
+    default FoodSelection foodSelection() {
+        return FoodSelection.SHARK;
```
Why not use Rs2Food here? It will include alot more options to allow the user to pick whatever food they want..

changednamedes replied:
Forgot all about it and I changed the food a while ago hah. Updated now to use Rs2Food. There is alot of options now though.

changednamedes committed:
Updated Scurrius

Added more config options for food and prayer, with min and max to use

Fixed the prayers to be used correctly and disable/enable correctly

fixed dodging of the dangerous tiles

Commit:
added option for public or private instance
